wondrous-dev/wondrous-frontend
# initial frontend changes to show templates to task creator

Frontend changes to add template tasks. 

Button found on CreateEntityModal below all the task specifications. Click to get two options: save or open options

Testing: 
- fill out task as you would normally do 
- click on templates 
- press save template and add a name in the text field that pops up 
- press the save button (should navigate to current templates)
- X out of the entire task modal and create a new one that is empty 
- go to templates and click open modal 
- Press the first one (should input all of the features in) 
- edit some of them 
- open the task template options modal 
- click on the ellipses and press edit. (should edit the frontend on spot) 
- then try deleting
